app/javascript/mastodon/initial_state.js

```diff
@@ -117,4 +117,11 @@ export const criticalUpdatesPending = initialState?.critical_updates_pending;
 export const statusPageUrl = getMeta('status_page_url');
 export const sso_redirect = getMeta('sso_redirect');
 
+/**
+ * @returns {string | undefined}
+ */
+export function getAccessToken() {
+  return getMeta('access_token');
+}
```

Comment:
Is there a reason this is a method and not a constant?

Reply:
No specific reason except allowing it to be retrieved another way if we want to change this at some point